core/src/test/scala/integration/kafka/server/QuorumTestHarness.scala

```diff
@@ -386,6 +386,7 @@ abstract class QuorumTestHarness extends Logging {
   def tearDown(): Unit = {
     Exit.resetExitProcedure()
     Exit.resetHaltProcedure()
+    TestUtils.clearYammerMetrics()
```

Comment:
Actually, should we do this after the implementation has shutdown? I don't know if the metrics would be used during shutdown